.github/workflows/build-rust-client.yml

```
@@ -1,75 +1,75 @@
+              uses: metaplex-foundation/actions/cache-crate@v1
+              with:
+                  folder: ./clients/rust
+                  key: rust-client-test
 
-      - name: Run cargo clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --all-targets --all-features --no-deps --manifest-path ./clients/rust/Cargo.toml
+            - name: Run cargo clippy
+              uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)